src/routes/MapExplorer/MapExplorer.js

```diff
@@ -274,13 +307,15 @@ export default class MapExplorer extends React.Component {
       <div styleName={cx('featureTypeFilters', { 'featureFiltersOpen': showFeatureFilters })}>
         <h3>What do you want to see on the map?</h3>
         {['member', 'request', 'offer', 'resource', 'event'].map(featureType => {
+          let color = FEATURE_TYPES[featureType].primaryColor
+
           return <div
             key={featureType}
             ref={this.refs[featureType]}
             styleName='featureTypeSwitch'
           >
             <SwitchStyled
-              backgroundColor={FEATURE_TYPES[featureType].primaryColor}
+              backgroundColor={`rgba(${color[0]}, ${color[1]}, ${color[2]}, ${color[3]/255})`}
```

Comment:
lint error here: `error  Infix operators must be spaced  space-infix-ops`

Reply:
fixed